packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/VirtualizedTraceView.test.js

```diff
@@ -373,6 +373,23 @@ describe('<VirtualizedTraceViewImpl>', () => {
       expect(spanBarRow.length).toBe(1);
       expect(spanBarRow.prop('noInstrumentedServer')).not.toBeNull();
     });
+
+    it('renders a SpanBarRow with a producer span and no instrumented server span', () => {
```

Comment:
This test is nearly identical to the one above. This makes it harder to maintain in sync. Can you not modify the existing test to loop through client/producer kinds and keep the rest the same?

Reply:
@yurishkuro Done, let me know if you want anything changed. Also, do you want me to combine `isKindProducer` and `isKindClient` into one `isKindProducerOrClient` as these are also quite similar.